python/cudf/cudf/tests/series/test_datetimelike.py

```diff
@@ -1 +1,113 @@
 # Copyright (c) 2023, NVIDIA CORPORATION.
+
+import os
+
+import pandas as pd
+import pytest
+
+import cudf
+from cudf import date_range
+from cudf.testing._utils import assert_eq
+
+
+def _get_all_zones():
+    zones = []
+    for root, dirs, files in os.walk("/usr/share/zoneinfo"):
```

Comment:
Just FYI, we ran into issues with `/etc/localtime` not existing in https://github.com/rapidsai/mambaforge-cuda/pull/16. The relation here is that I'm not 100% sure if you can assume `/usr/share/zoneinfo` is populated with data in super-minimalist Docker containers. What this means is that the result of this function could be an empty list. If you want that to raise an error/warning, you might want to add something explicit for that. I'm content with the current behavior or a warning. I might lean away from raising an error, but I'm not sure.

Reply:
No strong feelings. Let's leave it as-is and not overengineer it just to get a skip message.